ajax/libs/overpass/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "overpass",
   "version": "3.0.2",
-  "filename": "overpass-webfont/overpass.css",
+  "filename": "overpass-webfont/overpass.min.css",
```

Comment:
Hey @pvnr0082t need some help with that. I have two commits in total. How do I modify the first commit and make the change. I can't use `git rebase -i origin/add/overpass` and make the filename change because that would create a merge conflict with the second commit (I changed the filename originally in the second commit).